Components/list/list.css

```diff
@@ -0,0 +1,49 @@
+.list-container{
+    margin: 2rem 2rem 2rem 2rem;
+}
+
+.ordered-list{
+    font-family: poppin;
+    font-size: 1.5em;
+}
+
+.unordered-list{
+    font-family: poppin;
+    font-size: 1.5em;
```

Comment:
you are applying same properties to both classes so you can merge it using `ordered-list ,unordered-list{
font-family: poppin;
    font-size: 1.5em;
}
`